src/templates/zh-TW/2020/methodology.html

```diff
@@ -0,0 +1,476 @@
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+              <th>配置</th>
+              <th>桌機</th>
+              <th>手機</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>設備</td>
+            <td>Linux VM</td>
+            <td>模擬Moto G4</td>
+          </tr>
+
+          <tr>
+            <td>使用者代理</td>
+            <td>
+              Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/537.36 PTST/200805.230825
+            </td>
+            <td>
+              Mozilla/5.0 (Linux; Android 6.0.1; Moto G (4) Build/MPJ24.139-64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.146 Mobile Safari/537.36 PTST/200815.130813
+            </td>
+          </tr>
+
+          <tr>
+            <td>位置</td>
+            <td>
+              Redwood City, California, USA<br>
+              The Dalles, Oregon, USA
+            </td>
+            <td>
+              Redwood City, California, USA<br>
+              The Dalles, Oregon, USA
+            </td>
+          </tr>
+
+          <tr>
+            <td>連接</td>
+            <td>Cable (5/1 Mbps 28ms RTT)</td>
+            <td>3G (1.600/0.768 Mbps 300ms RTT)</td>
+          </tr>
+
+          <tr>
+            <td>Viewport</td>
+            <td>1024 x 768px</td>
+            <td>512 x 360px</td>
+          </tr>
+        </tbody>
+      </table>
+      </div></div>
+
+      <p>
+        桌機版網站是在 Linux 虛擬機的 Chrome 環境中運行，網路速度相當於有線連接。
+      </p>
+
+      <p>
+        手機版網站在模擬的 Moto G4 設備上運行，網路速度相當於 3G 連結。備註，模擬的手機版的使用者代理標示為 Chrome 65 ，但實際的瀏覽器是 Chrome 84。
+      </p>
+
+      <p>
+        測試運行的位置有二個站點： California 和 Oregon USA。 HTTP Archive 維護它自己的測試代理硬體，位置在 California 的<a href="https://archive.org"> Internet Archive </a>數據中心。另外一個測試代理是依據需要擴增，在<a href="https://cloud.google.com/compute/docs/regions-zones/#locations"> Google Cloud Platform </a>位於 Oregon 的 us-west-1。
+      </p>
+
+      <p>
+        HTTP Archive 在 WebPageTest 的私有實例保持與最新的公共版本同步，且增加使用<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">自訂指標</a>。在測試的尾聲透過片段的 JavaScript 評估每個網站。感謝許多數據分析的<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢獻者</a>，特別是<a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls?q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc">熱血拋顱</a>的<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito') }}"> Tony McCreath </a>。 Web Almanac 網路年鑑2020年版致力拓展 HTTP Archive 的基礎測試架構功能，揮灑超過三千行的新程式碼。
+      </p>
+
+      <p>
+        每個測試的結果都保存為<a href="https://en.wikipedia.org/wiki/HAR_(file_format)"> HAR 檔案</a>，這是 JSON 格式儲存的檔案，包括網頁的位元數據。
```

Comment:
```suggestion
        每個測試的結果都保存為<a hreflang="en" href="https://en.wikipedia.org/wiki/HAR_(file_format)">HAR 檔案</a>，這是 JSON 格式儲存的檔案，包括網頁的位元數據。
```

Reply:
Yes in this case we should change the link to Chinese version and therefore do not need to include the `hreflang`

Reply:
And actually for wikipedia we automatically recognise english here so wouldn't need `hreflang` even if we didn't have a Chinese version. Sorry forgot about that.

Reply:
thanks the well explanation. The link of Chinese wikipedia is updated to `href`.